README.md

```diff
@@ -7,6 +7,8 @@ This project very much under active development, and is currently not as reusabl
 
 ## Configuring for Development
 
+Make sure an instance of elasticsearch is available at `http://elasticsearch.local:9200/`. If you are running theonion/onion-services you can add `elasticsearch.local` to your `/etc/hosts` line that points at that docker dev machine.
+
```

Comment:
I'm OK with this -- onion-services will be running for most everyone, and /etc/hosts is an easy workaround. In the end it streamlines dev setup -- no reason to have ES running natively when it's also running as a container.